docs/standard/garbage-collection/implementing-disposeasync.md

```diff
@@ -127,6 +127,8 @@ If an exception is thrown from the `AnotherAsyncDisposable` constructor, then `o
 
 ## See also
 
+For a dual implementation example of `IDisposable` and `IAsyncDisposable`, see the <xref:System.Text.Json.Utf8JsonWriter> source code [on GitHub](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.cs#L297-L345).
```

Comment:
The line numbers can get outdated quickly when referencing master.

```suggestion
For a dual implementation example of `IDisposable` and `IAsyncDisposable`, see the <xref:System.Text.Json.Utf8JsonWriter> source code [on GitHub](https://github.com/dotnet/runtime/blob/035b729d829368c2790d825bd02db14f0c0fd2ea/src/libraries/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.cs#L297-L345).
```

Reply:
Good catch